panel/widgets/tables.py

```diff
@@ -1054,6 +1054,7 @@ def _process_event(self, event):
                 event.old = self._old[event.column].iloc[event.row]
             for cb in self._on_edit_callbacks:
                 cb(event)
+            self._update_style()
```

Comment:
Surprised this is needed. An edit event triggers two separate events 1) the `table-edit` event and 2) a property change event on the data source. I would expect the styles to already be updated by the latter.

Reply:
Commenting this doesn't resolve the issue so it appears it's needed. The data source update doesn't trigger any restyling apparently.